### PR TITLE
some questions - answers needed

### PR DESCRIPTION
**Pull request by clifle**
First draft of the FAQs page

### CODE REVIEW

File: src/main/paradox/docs/additional-info/faq.md [hunk 1]
```diff
@@ -1,3 +1,124 @@
...
+**What are the clients I can use with Nexus? What are the requirements to run Nexus locally?**
+
+On macOS and Windows, Docker effectively runs containers inside a VM created by the system hypervisor. Nexus requires at least 2 CPUs and 8 GiB of memory in total. You can increase the limits in Docker settings in the menu *Preferences > Advanced*. Further information [here](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/docker.html).
+
+
+**Where is the sandbox deployed?**
+
+
+**What is a SHACL schema?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: SHACL (Shapes Constraint Language) is a language for validating [RDF graphs](https://en.wikipedia.org/wiki/Resource_Description_Framework) against a set of conditions. These conditions are provided as shapes and other constructs expressed in the form of an RDF graph.
SHACL is used in Nexus to constrain and control the payload that can be pushed into Nexus.

File: src/main/paradox/docs/additional-info/faq.md [hunk 2]
```diff
@@ -1,3 +1,124 @@
...
+
+
+**How does SHACL compares with JSON Schema?**
+
+
+
+**Do I need to define SHACL schemas to bring data in?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: No. SHACL schemas provides with an extra layer of quality control for the data that is ingested into Nexus. However we acknowledge the complexity of defining schemas. That's why clients can decide whether to use schemas to constrain their data or not, depending on their use case and their available resources.

File: src/main/paradox/docs/additional-info/faq.md [hunk 3]
```diff
@@ -1,3 +1,124 @@
...
+
+
+**Where can I find SHACL shapes I can reuse (point to resources, like schema.org)?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: https://github.com/INCF/neuroshapes

File: src/main/paradox/docs/additional-info/faq.md [hunk 4]
```diff
@@ -1,3 +1,124 @@
...
+
+
+**What is JSON-LD?**
+
+JSON-LD is a JavaScript Object Notation for Linked Data. It is a set of W3C standards specifications which enable a semantic-preserving data exchange and allows to solve the ambiguity problem including a @context object, where every key is associated with an identifier. In this way, the object properties are linked to contexts in a JSON document to concepts in ontologies. 
+JSON-LD is fully compatible with the RDF data model.
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: JSON-LD is the main format we use for messages exchange. A JSON-LD payload is then converted to an RDF Graph for validation purposes and for ingestion in the Knowledge Graph.

File: src/main/paradox/docs/additional-info/faq.md [hunk 5]
```diff
@@ -1,3 +1,124 @@
...
+The Resource Description Framework (RDF) is a graph-based data model used for representing information in the Web. The basic structure of any expression in RDF is in triples, an extremely easy segmentation of any kind of knowledge in subject-predicate-object. It is a family of W3C specifications, and was originally design as a metadata model.
+ 
+**What is Elasticsearch?**
+
+Elasticsearch is a document oriented search engine with an HTTP web interface and schema-free JSON document. It’s able to aggregate data based on specific queries enabling the exploration of trends and pattern.
+
+
+**Why are RDF and JSON-LD important for Nexus?**
+
+Nexus is providing (for free) a REST HTTP API, an Elasticsearch indexation and a SPARQL endpoint if you already have RDF data.
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: RDF is the data model used to ingest data into the Knowledge Graph. RDF is also used for SHACL schema data validation

File: src/main/paradox/docs/additional-info/faq.md [hunk 6]
```diff
@@ -1,3 +1,124 @@
...
+
+
+**Can I connect any SPARQL client to Nexus’ SPARQL endpoint?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: As long as the client supports the ability to provide a `Authentication` HTTP Header (for authentication purposes) on the request, any SPARQL client should work.

File: src/main/paradox/docs/additional-info/faq.md [hunk 7]
```diff
@@ -1,3 +1,124 @@
...
+
+
+**Is there a cloud deployment of Nexus?**
+
+There is a sandbox for you to try Nexus, it has limited resources and is regularly wiped out. Further information [here](https://bluebrain.github.io/nexus/docs/webapps/index.html)
+
+
+**Is there a limit on the number of resources Nexus can store?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: Limitations and performance depends heavily on the deployment setup where Nexus is running. 

To get an idea about the ingestion capabilities, we have run [Benchmarks](https://bluebrainnexus.io/docs/additional-info/benchmarks/index.html) where we were able to ingest more than 115 million resources.

File: src/main/paradox/docs/additional-info/faq.md [hunk 8]
```diff
@@ -1,3 +1,124 @@
...
+Yes, Nexus is a free, Open Source platform released under [Apache Licence 2.0](https://opensource.org/licenses/Apache-2.0)
+
+
+**How to run Nexus?**
+
+There are many ways to run Nexus: a public instance is running at the [Sandbox](https://nexus-sandbox.io/web), meanwhile if you want to run it locally you might need to install [Docker](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/docker.html) or [Minikube](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/minikube.html). You can also deploy Nexus [“on premise”](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/minikube.html), as a single instance or as a cluster.
+
+
+**How can I try Nexus without installing it? What is the difference with a regular database like PostgreSQL?**
+
```
Comment (umbreak) [inline src/main/paradox/docs/additional-info/faq.md]: You can use the [cloud deployment](https://bluebrain.github.io/nexus/docs/webapps/index.html).
Although Nexus can be used as a regular database, it's flexibility and feature set are well beyond that. Just to mention some of the Nexus features:
- Allows to define different constraints to different set of data at runtime.
- Provides automatic indexing into several indexers (currently ElasticSearch and Sparql), dealing with reindexing strategies, retries and progress.
- Provides authentication
- Comes with a flexible and granular authorization mechanism.
- Guarantees resources immutability, keeping track of a history of changes.

Comment (clifle) [reply src/main/paradox/docs/additional-info/faq.md]: In cloud deployment link did you mean this https://bluebrainnexus.io//docs/getting-started/webapps.html or this https://bluebrainnexus.io//docs/getting-started/running-nexus/index.html#on-premise-cloud-deployment ?

Comment (umbreak) [reply src/main/paradox/docs/additional-info/faq.md]: This one: https://bluebrainnexus.io//docs/getting-started/webapps.html

File: src/main/paradox/docs/additional-info/faq.md (hunk 5 above)
Comment (clifle) [reply src/main/paradox/docs/additional-info/faq.md]: what about JSON-LD ?

Comment (umbreak) [reply src/main/paradox/docs/additional-info/faq.md]: Maybe let's just move my addition to the `What is Json-LD?` section here:

JSON-LD is the main format used in Nexus for messages exchange. A JSON-LD it is converted to an RDF Graph for validation purposes and for ingestion in the Knowledge Graph. The choice of JSON-LD is due to the fact that is as plain JSON but with some special [keywords](https://json-ld.org/spec/latest/json-ld/#syntax-tokens-and-keywords). And JSON is the preferred format for message exchanges for APIs.

File: src/main/paradox/docs/additional-info/faq.md [hunk 9]
```diff
@@ -1,3 +1,121 @@
...
+Yes. As long as the client supports the ability to provide a ```Authentication``` HTTP Header (for authentication purposes) on the request, any SPARQL client should work.
+
+
+**How I can create a Organizations as an anonymous user in the docker-compose file? What needs to be done to switch to "authenticated" mode?**
+
+The permissions for anonymous are preset in the [ACLs](https://bluebrainnexus.io/docs/api/1.0/iam/iam-permissions-api.html) and should be replaced by the standard authentication. More details [here](https://bluebrainnexus.io/docs/api/1.0/iam/iam-permissions-api.html).
+
+
+**Can I use Nexus from Jupyter Notebooks?**
```
Comment (wwajerowicz) [inline src/main/paradox/docs/additional-info/faq.md]: Nexus can be used from Jupyter Notebooks using [Nexus Python SDK](https://github.com/BlueBrain/nexus-python-sdk/). Alternatively, you can also use any Python HTTP client and use Nexus REST API directly from the Jupyter Notebook.

Comment (umbreak) [reply src/main/paradox/docs/additional-info/faq.md]: Maybe we can add some examples of jupyter notebooks that use nexus?
I think that is one: [Building a Data Pipeline for a Recommender System](https://github.com/BlueBrain/nexus/blob/master/src/main/paradox/docs/tutorial/notebooks/Recommendation%20System%20via%20Nexus.ipynb)

Comment (wwajerowicz) [reply src/main/paradox/docs/additional-info/faq.md]: I think these ones are also  [good examples](https://github.com/BlueBrain/nexus-python-sdk/tree/master/notebooks).

**Pull request closed without merging**